blocks.js

```diff
@@ -9584,6 +9705,15 @@ SymbolMorph.prototype.setLabelColor = function (
     this.setColor(textColor);
 };
 
+// SymbolMorph to "scratchblocks"
+
+SymbolMorph.prototype.stringify = function () {
+    var symbol = {
```

Comment:
I get what it's doing, but only after reading it like a few times. That reason makes sense, though it seems better, to me, to  add complexity as needed, that's all.